lib/flutter_svg_provider.dart

```diff
@@ -122,6 +122,15 @@ class Svg extends ImageProvider<SvgImageKey> {
   // [SvgImageKey] instances will be compared instead.
   @override
   String toString() => '$runtimeType(${describeIdentity(path)})';
+  
+  // Running on web with Colors.transparent may throws the exception `Expected a value of type 'SkDeletable', but got one of type 'Null'`.
+  static Color getFilterColor(color) {
+    if (kIsWeb && color == Colors.transparent) {
+      return const Color(0x01ffffff);
+    } else {
+      return color;
```

Comment:
In this case, should return `Colors.transparent` if color is `null`.

Reply:
In fact,`key.color` has been Initialized in function `obtainKey` and defined with `final`,so color can't be `null`.But for a better reading experience,I agree with you.   :)